### ISSUE TITLE
Sign bug

### ISSUE THREAD

**Issue by caligero3000** [dalaranwow/dalaran-wow#5035]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Sign south of The Sepulcher is looking weird. The sign that shows the players the way to the Deep Elem Mine is disconnected from the signpost and is currently just hanging in mid-air.

**Expected behaviour**: Fix pls.

**Steps to reproduce the problem**:

1. Go and
2. Fix
3. It

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello @caligero3000,
Incomplete reports will be closed as I prefer to spent my time on something useful instead of guessing what you are tring to say here.
Please fill the entire templete and the report will be revisited.
